templates/google_one_tap_button.html

```
@@ -1,5 +1,6 @@
-<span class="pe-2" style="height: 44px; width: 200px" data-replace-login-spinner>
-    <span id="g_id_signin"></span>
+<span class="pe-2" style="height: 44px; max-width: 200px" data-replace-login-spinner>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)